pyproject.toml

```diff
@@ -10,9 +10,9 @@ license = "MPL-2.0"
 [tool.poetry.dependencies]
 python = "^3.9"
 numpy = "1.26.4"
-pypowsybl = "^1.9.0"
-grid2op = "^1.10.5"
+pypowsybl = "1.11.0"
 sortedcontainers = "^2.4.0"
+grid2op = "1.11.0.dev5"
```

Comment:
I am not sure if we force grid2op 1.11.0 here. The code should work with grid2op < 1.11, doesn't it ?

Reply:
Yes, reverted to 1.10.5